cocos/ui/UIEditBox/UIEditBoxImpl-common.cpp

```diff
@@ -145,7 +145,7 @@ void EditBoxImplCommon::setFontColor(const Color4B& color)
 
 void EditBoxImplCommon::setPlaceholderFont(const char* pFontName, int fontSize)
 {
-    this->setNativePlaceholderFont(pFontName, fontSize);
+    this->setNativePlaceholderFont(pFontName, fontSize * _labelPlaceHolder->nodeToWorldTransform().a);
```

Comment:
`nodeToWorldTransform` is a deprecated method, please use `getNodeToWorldAffineTransform`  instead.